### PR TITLE
RCHAIN-3113: Add curly brackets around receive in match case when pretty printing

### PR DESCRIPTION
**Pull request by tymm**
## Overview
Without this fix the PrettyPrinter outputs Rholang that is not parsable.

Another solution might be to change the grammar in `rholang_mercury.cf` from
```
-- Match Cases
CaseImpl. Case ::= Proc13 "=>" Proc3 ;
```
to
```
-- Match Cases
CaseImpl. Case ::= Proc13 "=>" Proc1 ;
```
This would result in `match Nil { _ => for(_ <- @Nil) {Nil} }` to be parsable.
I didn't go that route because it looks to me as if there is *some* reason that the bnfc rule is what it is.

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3113

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala [hunk 1]
```diff
@@ -745,6 +745,18 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
...
 
+  it should "Print receive in curly brackets" in {
+    checkRoundTrip(
+      """match Nil {
+        |  _ => {
+        |    for( @{_} <- @{Nil} ) {
+        |      Nil
+        |    }
+        |  }
+        |}""".stripMargin
```
Comment (ArturGajowy) [inline rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: We should actually always put braces around the process that is the body of a match case

Comment (tymm) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: Except for sends, right?

Comment (tymm) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: And out of interest: why do you think so?

Comment (ArturGajowy) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: why except for sends?
I think so because of consistency :)

Comment (ArturGajowy) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: And implementation ease

Comment (ArturGajowy) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: and then, maintainance ease (I really hope we'll rewrite the pretty-printer in the future)

Comment (tymm) [reply rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala]: Nevermind, it was late. I somehow thought you were talking about the grammar and not the PrettyPrinter.

**Pull request merged**